### PR TITLE
minor fitting to set versions for our operator

### PR DESCRIPTION
**Pull request by mfojtik**
https://github.com/openshift/cluster-kube-apiserver-operator/pull/308

### CODE REVIEW

File: manifests/0000_61_openshift-apiserver-operator_03_version-mapping.yaml [hunk 1]
```diff
@@ -4,5 +4,5 @@ metadata:
   namespace: openshift-apiserver-operator
   name: version-mapping
 data:
-  "0.0.0_version_hypershift": "quay.io/openshift/origin-hypershift:v4.0"
-  "0.0.0_version_cluster-openshift-apiserver-operator": "docker.io/openshift/origin-cluster-openshift-apiserver-operator:v4.0"
+  "0.0.1-snapshot-kubernetes": "quay.io/openshift/origin-hypershift:v4.0"
```
Comment (deads2k) [inline manifests/0000_61_openshift-apiserver-operator_03_version-mapping.yaml]: openshift, not kube

Comment (mfojtik) [reply manifests/0000_61_openshift-apiserver-operator_03_version-mapping.yaml]: fixed. also in clusteroperator

File: manifests/0000_61_openshift-apiserver-operator_03_version-mapping.yaml [hunk 2]
```diff
@@ -4,5 +4,5 @@ metadata:
   namespace: openshift-apiserver-operator
   name: version-mapping
 data:
-  "0.0.0_version_hypershift": "quay.io/openshift/origin-hypershift:v4.0"
-  "0.0.0_version_cluster-openshift-apiserver-operator": "docker.io/openshift/origin-cluster-openshift-apiserver-operator:v4.0"
+  "0.0.1-snapshot-openshift": "quay.io/openshift/origin-hypershift:v4.0"
```
Comment (smarterclayton) [inline manifests/0000_61_openshift-apiserver-operator_03_version-mapping.yaml]: You can't do this.  We don't have a version for openshift.

File: manifests/0000_61_openshift-apiserver-operator_08_clusteroperator.yaml [hunk 3]
```diff
@@ -3,3 +3,9 @@ kind: ClusterOperator
 metadata:
   name: openshift-apiserver
 spec: {}
+status:
+  versions:
+    - name: operator
+      version: "0.0.1-snapshot"
+    - name: openshift-apiserver
+      version: "0.0.1-snapshot-openshift"
```
Comment (smarterclayton) [inline manifests/0000_61_openshift-apiserver-operator_08_clusteroperator.yaml]: Will not work.  You would just report `0.0.1-snapshot` here.

File: manifests/0000_61_openshift-apiserver-operator_03_version-mapping.yaml (hunk 2 above)
Comment (smarterclayton) [reply manifests/0000_61_openshift-apiserver-operator_03_version-mapping.yaml]: Would suggest `openshift-0.0.1-snapshot` or `0.0.1-snapshot_openshift` depending on your use case.

**Comment by mfojtik** [comment]
@smarterclayton @deads2k PTAL

**Pull request merged**